mkimage

```diff
@@ -164,8 +164,25 @@ cat > "$rootfsDir/usr/sbin/install_packages" <<-'EOF'
 set -e
 set -u
 export DEBIAN_FRONTEND=noninteractive
-apt-get update -qq
-apt-get install -y --no-install-recommends "$@"
+n=0
+max=2
```

Comment:
should we use an environment variable here (with a default value)? that way we can increase the retries in certain environments